src/services/side-panels-manager/models/item-wrapper.tsx

```diff
@@ -27,7 +27,6 @@ export class ItemWrapper {
   public activate(): void {
     if (this.panelItemComponentRef.current) {
       this.panelItemComponentRef.current!.on();
-      this.item.onActivate?.();
```

Comment:
so now that the calls are removed, how does kava plugin know when a side panel is activated / deactivated ?